### PR TITLE
feat: add support for multiple .env files

### PR DESCRIPTION
**Pull request by AkhilJ321**
**Description**

- Added Support for multiple .env files.
- Added Documentation for the same.

**Related issue(s)**
Fixes: #638

### CODE REVIEW

**Comment by Souvikns** [comment]
@AkhilJ321 you have some linter issues, can you double-check it once thanks.

**Comment by AkhilJ321** [comment]
> @AkhilJ321 you have some linter issues, can you double-check it once thanks.

Handled the linting errors,also corrected the documentation .

File: docs/pages/env-vars-config.md [hunk 1]
```diff
@@ -13,6 +13,15 @@ Glee provides a few environment variables for you to customize the Glee applicat
 |GLEE_SERVER_CERTS|A comma-separated list of `${serverName}:${pathToCertificateFile}`. These are the certificates to use when establishing the connection to the given server.|`GLEE_SERVER_CERTS=mosquitto:mosquitto.org.crt`|
 |GLEE_SERVER_VARIABLES|A comma-separated list of `${serverName}:${serverVariable}:${value}`. These are the values to use for each server variable.|`GLEE_SERVER_VARIABLES=websockets:namespace:public`|
 
+### Support for multiple .env files
+Glee has support for loading variables from `.env.local` into `process.env`.
+This is useful for storing secret environment variables needed in development while keeping them out of the repository.
+
+However, sometimes you might want to add some defaults for the `development` or `production` environment. You can do that by creating files with the following names:
```
Comment (KhudaDad414) [inline docs/pages/env-vars-config.md:20]: please mention how the user can change the environment. (by setting NODE_ENV in their environment.)

**Review by KhudaDad414 — CHANGES_REQUESTED** [review]

**Comment by AkhilJ321** [comment]
I had some issues with github , and my forked repo got all unbalanced. So I will create a new PR after creating a new fork and making changes.

**Pull request closed without merging**